Common/GPU/OpenGL/thin3d_gl.cpp

```diff
@@ -534,6 +534,7 @@ OpenGLContext::OpenGLContext() {
 	caps_.framebufferBlitSupported = gl_extensions.NV_framebuffer_blit || gl_extensions.ARB_framebuffer_object;
 	caps_.framebufferDepthBlitSupported = caps_.framebufferBlitSupported;
 	caps_.depthClampSupported = gl_extensions.ARB_depth_clamp;
+	caps_.clipCullDistanceSupported = gl_extensions.EXT_clip_cull_distance || (!gl_extensions.IsGLES && gl_extensions.VersionGEThan(3, 0));
```

Comment:
`gl_CullDistance` is part of [OpenGL 4.5](https://www.khronos.org/registry/OpenGL-Refpages/gl4/html/gl_CullDistance.xhtml), not 3.0.

```suggestion
	caps_.clipCullDistanceSupported = gl_extensions.EXT_clip_cull_distance || (!gl_extensions.IsGLES && gl_extensions.VersionGEThan(4, 5));
```